src/main/java/com/cognifide/maven/plugins/vltco/goals/clean/CleanHelper.java

```diff
@@ -48,7 +51,13 @@ public static void cleanupDotContent(String root, String[] contentProperties) th
 						lines.add(line);
 					}
 				}
-				FileUtils.writeLines(file, CharEncoding.UTF_8, lines);
+				String line = null;
```

Comment:
looks like a good candidate for a private function i.e. 'getLineEnding' or something like that